doc/base/classes.xml

```diff
@@ -12217,7 +12217,11 @@
 			<argument index="3" name="a" type="float">
 			</argument>
 			<description>
-				Construct the color from an RGBA profile.
+				Constructs a color from an RGBA profile using values between 0 and 1 (float).
+
```

Comment:
Could you amend the commit to remove the empty lines before code blocks? Those will be parsed as an empty paragraph in the docs, adding useless spacing.